autogen/main/variables.tf.tmpl

```diff
@@ -242,30 +242,34 @@ variable "cluster_autoscaling" {
 {% if beta_cluster %}
     autoscaling_profile = string
 {% endif %}
-    min_cpu_cores       = number
-    max_cpu_cores       = number
-    min_memory_gb       = number
-    max_memory_gb       = number
-    gpu_resources       = list(object({ resource_type = string, minimum = number, maximum = number }))
-    auto_repair         = bool
-    auto_upgrade        = bool
-    disk_size           = optional(number)
-    disk_type           = optional(string)
+    max_cpu_cores               = number
+    min_memory_gb               = number
+    min_cpu_cores               = number
+    max_memory_gb               = number
+    gpu_resources               = list(object({ resource_type = string, minimum = number, maximum = number }))
+    auto_repair                 = bool
+    auto_upgrade                = bool
+    disk_size                   = optional(number, 100)
+    disk_type                   = optional(string, "pd-standard")
```

Comment:
the default values for optional needs to be provided in the modifier, otherwise, terraform will still require the attribute.